examples/src/com/messente/examples/simple/GetAccountBalanceSimpleExample.java

```diff
@@ -37,6 +40,8 @@ public static void main(String[] args) {
         MessenteResponse response = null;
 
         try {
+            // #### EXAMPLE 1 ####

```

Comment:
```suggestion
            // #### EXAMPLE 1 ####
```suggestion
            // #### EXAMPLE 1 ####
**[]()**
```

```